tests/integration_tests/test_temporal_data_tasks.py

```diff
@@ -72,6 +72,38 @@ def test_temporal_classification_functional():
     assert(history.history['acc'][-1] >= 0.8)
 
 
+def test_temporal_classification_functional_local_random_seed():
```

Comment:
This integration test isn't really testing the random state.

Please remove the added test. The PR as it is is simple enough that it doesn't warrant testing.

Reply:
Done